<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AIP61047062S</string>
  </property>
  <property name="toolTipDuration">
   <number>-100</number>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QComboBox" name="file_comboBox">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>F i l e</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Input Image</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Save Image</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="function_comboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="accessibleDescription">
        <string notr="true"/>
       </property>
       <item>
        <property name="text">
         <string>Function</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Return Original Image</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>To Histogram</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gaussian Noise</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wavelet Transform</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Histogram Equalization</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Convolution</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Binary</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dilation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Erosion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>License plate recognition</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ïƒ:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Show GN Histogram</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>771</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Original histogram</string>
       </property>
       <widget class="QLabel" name="originalhistogram">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>351</width>
          <height>351</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>27</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Original Histogram</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Processing histogram</string>
       </property>
       <widget class="QLabel" name="processinghistogram">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>351</width>
          <height>351</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Processing Histogram</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>771</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Original Image</string>
       </property>
       <widget class="QLabel" name="originalimage">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>351</width>
          <height>351</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>27</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Original Image</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Processing Image</string>
       </property>
       <widget class="QLabel" name="processingimage">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>351</width>
          <height>351</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Processing Image</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>50</y>
      <width>271</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Convolution Kernel</string>
    </property>
    <widget class="QSpinBox" name="spinBox_40">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_31">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_04">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_03">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_21">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_44">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>150</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_34">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_43">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_13">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_30">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_00">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_11">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_41">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_01">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_33">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_42">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_24">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_14">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_22">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_23">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_02">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_12">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_32">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
    </widget>
    <widget class="QPushButton" name="h8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>240</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>h8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="h4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>h4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="h1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>h1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="h3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>h3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="h5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>h5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="h7">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>h7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="h6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>h6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="h2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>h2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="conv3_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>3x3 Conv</string>
     </property>
    </widget>
    <widget class="QPushButton" name="conv5_button">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>270</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>5x5 Conv</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>edge detection</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>smoothing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="x3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>210</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>3x3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="x5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>5x5</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
